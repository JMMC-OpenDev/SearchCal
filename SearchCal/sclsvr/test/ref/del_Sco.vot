version https://git-lfs.github.com/spec/v1
oid sha256:3de7a9a514f63416ecf1dbcd7ff9d9b1707a4979967f23ffde193b3f02ea50da
size 192763
