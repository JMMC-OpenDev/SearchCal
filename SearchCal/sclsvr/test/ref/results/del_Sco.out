version https://git-lfs.github.com/spec/v1
oid sha256:23a67b285281f7e24fbc4186a7c731a0134c1bd47c3c180ff772b0a33ceaf402
size 137936
